bgcval2/UKESMpython.py

```diff
@@ -71,8 +71,8 @@ def folder(name):
     """ This snippet takes a string, makes the folder and the string.
 	    It also accepts lists of strings.
 	"""
-    if type(name) == type(['a', 'b', 'c']):
-        name = join(name, '/')
+    if isinstance(name, list):
+        name = os.path.join(*name) # splat!
```

Comment:
```suggestion
        name = os.path.join(*name)
```

Reply:
I've never used splat before, so I added this comment to remind me what that operator does.